examples/app/Embeddings.ipynb

```diff
@@ -7,8 +7,26 @@
    "source": [
     "## Embedding explorer\n",
     "\n",
-    "- Embeds a string column using OpenAI Embeddings\n",
-    "- Follows the clustering / projection approach from here: https://umap-learn.readthedocs.io/en/latest/clustering.html"
+    "In this tutorial, we'll create a notebook-based Embeddings explorer. We will use functions, or `Ops`, defined in the weave ecosystem so that they're available to compatible types in the Weave UI.\n",
+    "\n",
+    "You'll learn how to:\n",
+    "- Save and version embeddings\n",
+    "- Build a interactive embeddings explorer using Weave\n",
+    "- Use the weave ecosystem to: \n",
+    "    - Embed a string column using OpenAI Embeddings\n",
+    "    - Cluster & project the embeddings using umap\n",
+    "\n",
+    "Note: we will follow the clustering / projection approach from here: https://umap-learn.readthedocs.io/en/latest/clustering.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dd81428f",
+   "metadata": {},
+   "source": [
+    "## Import Dependencies\n",
+    "\n",
+    "Make sure you have weave installed using `pip install weave`."
```

Comment:
looks like we also need the following—let's make sure they're mentioned in the notebook/maybe commented out as a cell in the setup/dependencies? otherwise the user will need to jump through these hoops
we need to pip install
- openai
- tiktoken
- tenacity
- hdbscan

Reply:
good catch, added in new PR